apps/ledger-live-desktop/static/i18n/en/app.json

```
@@ -3705,8 +3705,10 @@
       "debugAppsDesc": "Display and allow opening debug tagged platform apps.",
       "experimentalApps": "Allow experimental apps",
       "experimentalAppsDesc": "Display and allow opening experimental tagged platform apps.",
-      "catalogServer": "Set the catalog provider",
-      "catalogServerDesc": "Switch between multiple platform apps sources",
+      "catalogServer": "Set the catalog apps provider",
+      "catalogServerDesc": "Enter your custom catalog provider URL",
```

I think we can rename this keys to avoid matching the old trads on other languages
```suggestion
      "catalogServerUrl": "Set the catalog apps provider",
      "catalogServerUrlDesc": "Enter your custom catalog provider URL",
```
or
```suggestion
      "catalogUrl": "Set the catalog apps provider",
      "catalogUrlDesc": "Enter your custom catalog provider URL",
```